README.md

```diff
@@ -85,4 +89,4 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 
-A copy of the license is available in the repository's [license.txt]( https://raw.github.com/Esri/arcgis-runtime-samples-ios/master/license.txt) file.
+A copy of the license is available in the repository's [LICENSE](https://raw.githubusercontent.com/Esri/arcgis-runtime-samples-ios/master/LICENSE) file.
```

Comment:
Maybe this should link to the file on GitHub rather than the raw file?
```suggestion
A copy of the license is available in the repository's [LICENSE](https://github.com/Esri/arcgis-runtime-samples-ios/blob/master/LICENSE) file.
```